From d79979710e11fbb0955321ca274c561437f2927b Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Mon, 1 Apr 2019 19:45:28 +0900
Subject: [PATCH 792/909] Revert "arm64: dts: r8a7796: Connect SDHI to
 IPMMU-DS1"

commit 03d26176bf2fceb0c28511613a7b901ef4866f58 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This reverts commit 6ea36b7fb2cbf0b4ab5cf0d743de64a833468886.

Multiple IPMMUs can not be operated simultaneously due to M3 Ver.1.x
hardware restriction.

As a result, this reverts commit 6ea36b7fb2cb ("arm64: dts: r8a7796:
Connect SDHI to IPMMU-DS1") to keep IPMMU for SDHI{0..3} is disabled by
default.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7796.dtsi | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7796.dtsi b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
index 796d217e877d..e18b9a839063 100644
--- a/arch/arm64/boot/dts/renesas/r8a7796.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
@@ -2651,7 +2651,6 @@
 			max-frequency = <200000000>;
 			power-domains = <&sysc R8A7796_PD_ALWAYS_ON>;
 			resets = <&cpg 314>;
-			iommus = <&ipmmu_ds1 32>;
 			status = "disabled";
 		};
 
@@ -2664,7 +2663,6 @@
 			max-frequency = <200000000>;
 			power-domains = <&sysc R8A7796_PD_ALWAYS_ON>;
 			resets = <&cpg 313>;
-			iommus = <&ipmmu_ds1 33>;
 			status = "disabled";
 		};
 
@@ -2677,7 +2675,6 @@
 			max-frequency = <200000000>;
 			power-domains = <&sysc R8A7796_PD_ALWAYS_ON>;
 			resets = <&cpg 312>;
-			iommus = <&ipmmu_ds1 34>;
 			status = "disabled";
 		};
 
@@ -2690,7 +2687,6 @@
 			max-frequency = <200000000>;
 			power-domains = <&sysc R8A7796_PD_ALWAYS_ON>;
 			resets = <&cpg 311>;
-			iommus = <&ipmmu_ds1 35>;
 			status = "disabled";
 		};
 
-- 
2.17.1

